tests/src/Kernel/Entity/FieldAccessTest.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Kernel\Entity\FieldAccessTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\Core\Session\AnonymousUserSession;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Og;
+use Drupal\Component\Utility\Unicode;
+use Drupal\node\Entity\NodeType;
+use Drupal\user\Entity\User;
+use Drupal\user\Entity\Role;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Tests OG specific field access.
+ *
+ * @group og
+ */
+class FieldAccessTest extends KernelTestBase {
```

Comment:
Even if we have a trait, we will still want an `OgKernelTestBase` regardless, I think.